modules/ROOT/pages/mutations/update.adoc

```diff
@@ -199,12 +176,21 @@ type Movie {
 }
 ----
 
-We can pop from this `tags` property array.
+You can `POP` from this `tags` property array:
```

Comment:
Similar to my first comment around backticks - I think we should be consistent. Much like when we had "Query" and "Mutation" with the first capitalised, I think we should think about how the sentences read. I think saying "popping" something from an array is quite common.

```suggestion
You can pop from this `tags` property array:
```